conf/app.conf

```diff
@@ -143,7 +146,7 @@ log.error.output = stderr
 #  [dev] section for documentation of the various settings
 [prod]
 mode.dev = false
-
+server.host=${CC_SERVER_HOST}
```

Comment:
Is still used in the controller?
I saw that you provide ```revel.Config.StringDefault("server.host",host)``` for creating the URL, which would overwrite the env variable, correct?